Fabric.Authorization.Persistence.SqlServer/Fabric.Authorization.Persistence.SqlServer.csproj

```diff
@@ -21,7 +21,6 @@
   <ItemGroup>
     <Folder Include="DependencyInjection\" />
     <Folder Include="Configuration\" />
-    <Folder Include="Mappers\" />
```

Comment:
did the Mappers folder get excluded?

Reply:
Not sure why that keeps happening locally for me. I’ll fix.

Reply:
check the project file and make sure there is not an ItemGroup section that is removing it. I saw it in there before and removed it